lib/stdlib/doc/src/binary.xml

```diff
@@ -256,6 +256,22 @@
       </desc>
     </func>
 
+    <func>
+      <name name="encode_hex" arity="2"/>
+      <fsummary>Encodes a binary into a hex encoded binary with specified case</fsummary>
+      <desc>
+          <p>Encodes a binary into a hex encoded binary using the specified case for the Hex digits "a" to "f".</p>
```

Comment:
Done

Reply:
I have seen this style first in OTP Documentation. Could you please give the reference for the new style?  @Maria-12648430

Reply:
> Could you please give the reference for the new style?

Not sure if you can call it a "new" style, the possibility for it probably existed for quite a while. It is not used everywhere where it could be used, so I guess most of the documentation that was done before this became possible just didn't get reworked.

Anyway, [here in the Reference Manual DTDs](https://www.erlang.org/doc/apps/erl_docgen/refman_dtds.html#%3Cfunc%3E) it says that `<func>` "Contains one **or more** `<name>`...". A place where it is used is for example [the docs for `string:equal/2,3,4`](https://github.com/erlang/otp/blob/7ad716dfdbd674f96d4301cf6dab4f8791fbcdf8/lib/stdlib/doc/src/string.xml#L167-L169), which thereby looks [like this](https://github.com/erlang/otp/blob/7ad716dfdbd674f96d4301cf6dab4f8791fbcdf8/lib/stdlib/doc/src/string.xml#L167-L169) in the compiled documentation.